#1/template.md

```diff
@@ -16,7 +16,7 @@ What are you planning to achieve. Be as detailed as possible. Try to break it do
 ## Problem
 
 ## Solution
-
+An off ramping solution for Africans
```

Comment:
Could you provide more context? How is this different from the competitors? What problem are you solving that other on and offramps haven't solved yet?